dataset_metrics/Dataset_metrics.ipynb

```diff
@@ -411,6 +414,37 @@
    "source": [
     "instance_of_metrics.to_csv(output_dir+\"/\"+\"Number of articles with and without valid instance_of\")"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "chronic-clothing",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### Number of redirect articles \n",
+    "Validate that no \"page redirects\" are present in the dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "taken-ordinary",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "query = f\"\"\"\n",
+    "        select im.snapshot, count(*) as page_redirect from {username}.imagerec im\n",
```

Comment:
I templated the table name for this specific query to introduce a convention. It would be nice to template other queries too, so that we remove dependencies on my username/hdfs home. We can address this in a separate task. I did not want to introduce too many changes at once to this json blob.